packages/components/organisms/f-header/src/components/FlagIcon.vue

```diff
@@ -0,0 +1,68 @@
+<template>
+    <component :is="iconComponent" />
+</template>
+
+<script>
+import {
+    FlagAtRoundIcon as FlagIconAt,
```

Comment:
One thing it would be interesting to keep an eye on is the size of this package following the addition of more icons; I recently looked into why the f-footer is so large for such a small component and it was because of the icons which are all bundled up regardless of whether of not they are used in the current tenant (I tested by removing the icons and it went from ~108kb down to ~9kb which is a massive difference).

Reply:
I wonder if there would be a way of lazy-loading in these icons rather than including them inline, as I imagine it'll be a very small subset of users that will ever hover over or click the country selector.

Reply:
I did look at whether it was possible to use `import(` inside of an npm package and it _did_ generate an extra file for the icons, the issue is that that file then contains _all_ of the icons rather than just the ones referenced so I didn't take it any further.

Reply:
@DamianMullins @asherstoppard I will add a separate ticket for this under engineering excellence